### DISCUSSION TITLE
Corfu stealing home/end keys

### DISCUSSION THREAD

**Discussion by egr95** [minad/corfu#446] [Q&A] [answered]
Thanks for the awesome package!

This is minor but I see that when the corfu popup is active, `<home>` is mapped to `corfu-prompt-beginning` and `<end>` is mapped to `corfu-prompt-end`. I find this intrusive and would prefer to keep these keys mapped to their usual state. I tried all of the below but whenever the corfu popup becomes active again, these keys once again get stolen by corfu.

Thanks!

```
(define-key corfu-map "<home>" move-beginning-of-line)
(define-key corfu-map [home] nil)
(define-key corfu-map [remap corfu-prompt-beginning] nil)
```

**Comment by minad** [comment, marked as answer]
Did you try `(keymap-unset corfu-map "<remap> <move-end-of-line>")`?

**Comment by egr95** [comment]
That worked, thank you!
